lib/console.js

```diff
@@ -21,7 +21,7 @@ function Console(stdout, stderr) {
   Object.defineProperty(this, '_stdout', prop);
   prop.value = stderr;
   Object.defineProperty(this, '_stderr', prop);
-  prop.value = {};
```

Comment:
It might be better (perf-wise) to just replace this with `{ __proto__: null }` or `Object.create(null)`